_episodes_rmd/04-ggplot2.Rmd

```diff
@@ -509,7 +509,7 @@ The `labs` function takes the following arguments:
 - `...` -- any pair of name and value for aesthetics used in the plot (e.g., 
 `x`, `y`, `fill`, `color`, `size`)
 
-```{r barplot-wall-types-labeled}
+```{r barplot-wall-types-labeled, fig. alt = "Previous plot with plot title and labells added."}
```

Comment:
```suggestion
```{r barplot-wall-types-labeled, fig.alt = "Previous plot with plot title and labells added."}
```